aspnetcore/fundamentals/logging/index.md

```diff
@@ -41,62 +63,101 @@ If you use `CreateDefaultBuilder`, you can replace the default providers with yo
 
 ::: moniker-end
 
-::: moniker range="< aspnetcore-2.0"
+Learn more about [built-in logging providers](#built-in-logging-providers) and [third-party logging providers](#third-party-logging-providers) later in the article.
 
-To use a provider, install its NuGet package and call the provider's extension method on an instance of <xref:Microsoft.Extensions.Logging.ILoggerFactory>:
+## Create logs
 
-[!code-csharp[](index/samples/1.x/TodoApiSample/Startup.cs?name=snippet_AddConsoleAndDebug&highlight=3,5-7)]
+To create logs, get an <xref:Microsoft.Extensions.Logging.ILogger%601> object from dependency injection (DI).
 
-ASP.NET Core [dependency injection (DI)](xref:fundamentals/dependency-injection) provides the `ILoggerFactory` instance. The `AddConsole` and `AddDebug` extension methods are defined in the [Microsoft.Extensions.Logging.Console](https://www.nuget.org/packages/Microsoft.Extensions.Logging.Console/) and [Microsoft.Extensions.Logging.Debug](https://www.nuget.org/packages/Microsoft.Extensions.Logging.Debug/) packages. Each extension method calls the `ILoggerFactory.AddProvider` method, passing in an instance of the provider.
+The following Razor Pages example creates a logger with `TodoApiSample.Pages.AboutModel` as the *category*. 
```

Comment:
I would get rid of one of these examples and re-organize the section around the single example.